workshop/7 - Performance Efficiency/README.md

```diff
@@ -0,0 +1,207 @@
+# 7- Performance Efficiency
+
+Performance efficiency is the ability of a workload to scale and meet the demands placed on it by users in an efficient manner. In cloud environments, a workload should anticipate increases in demand to meet business requirements.
+
+## The Cache-Aside pattern
+
+The Cache-Aside pattern is a technique that's used to manage in-memory data caching. It reduces the request response time and can lead to increased response throughput. This efficiency reduces the number of horizontal scaling events, making the app more capable of handling traffic bursts. It also improves service availability by reducing the load on the primary data store and decreasing the likelihood of service outages.
+
+### Cache-Aside in Relecloud Concerts
```

Comment:
we may want to revisit this once .net 9 is released. at that point, the package IDistributedCache is in will have a new type called "HybridCache" that can handle two-tier caching. The package targets .net standard, so it's not dependent on updating tfms of the project